discourse_rock/patches/rake.patch

```diff
@@ -0,0 +1,87 @@
+--- /dev/null
++++ b/lib/tasks/discourse-charm.rake
+@@ -0,0 +1,88 @@
++# frozen_string_literal: true
++
++desc "Promote a user to admin"
++task "admin:promote", [:email] => [:environment] do |_, args|
++email = args[:email]
++  if !email || email !~ /@/
++    puts "ERROR: Expecting rake admin:promote[some@email.com]"
++    exit 1
++  end
++  if User.find_by_email(email).nil?
++    puts "ERROR: User with email #{email} not found"
++    exit 1
++  end
++ admin = User.find_by_email(email)
++ admin.grant_admin!
```

Comment:
What's this exclamation point at the end?

Reply:
upstream (discourse) method